fakemetrics.go

```diff
@@ -196,15 +196,17 @@ func runMultiplied(orgs, keysPerOrg, metricPeriod, flushPeriod, offset, speedup
 		for o := 1; o <= orgs; o++ {
 			for k := 1; k <= keysPerOrg; k++ {
 				i := (r-1)*orgs*keysPerOrg + (o-1)*keysPerOrg + k - 1
+				metric := "some.id.of.a.metric"
+				name := fmt.Sprintf("%s.%d", metric, k)
 				metrics[i] = &schema.MetricData{
-					Name:     fmt.Sprintf("some.id.of.a.metric.%d", k),
-					Metric:   "some.id.of.a.metric",
+					Name:     name,
+					Metric:   metric,
 					OrgId:    o,
 					Interval: metricPeriod,
 					Value:    0,
 					Unit:     "ms",
 					Mtype:    "gauge",
-					Tags:     []string{"some_tag", "ok", fmt.Sprintf("k:%d", k)},
+					Tags:     []string{"some=tag", fmt.Sprintf("name=%s", name), fmt.Sprintf("metric=%s", metric)},
```

Comment:
why tag every metric with `metric=some.id.of.a.metric` ? that doesn't seem very realistic. maybe tag it `fmt.Sprintf("id=%d",k)` instead

Reply:
ditto for name tag

Reply:
Regarding the `name` tag, that's what carbon does, so that's just consistent. Then regarding the `metric` tag, i basically just wanted to have some tags which refer to a small number of series and some that refer to a large number of series, because that makes testing different edge cases easier.